plugins/single-instance/src/lib.rs

```diff
@@ -35,7 +34,8 @@ pub fn init<R: Runtime, F: FnMut(&AppHandle<R>, Vec<String>, String) + Send + Sy
     mut f: F,
 ) -> TauriPlugin<R> {
     platform_impl::init(Box::new(move |app, args, cwd| {
-        if let Some(deep_link) = app.try_state::<DeepLink<R>>() {
+        #[cfg(feature = "deep-link")]
+        if let Some(deep_link) = app.try_state::<tauri_plugin_deep_link::DeepLink<R>>() {
```

Comment:
actually for this to work I believe the deep link plugin must be registered first - so the DeepLink state has been initialized

Reply:
though maybe that's not the case?? i remember it worked on linux idk

Reply:
This is only called when the second instance notified the first one, and it's run on the first instance which should have initialized all the plugins at this point, not sure if it's going to cause problem if we spawned the second instances too fast though